utils/wasm-builder/README.md

```diff
@@ -47,17 +47,17 @@ mod code {
 }
 
 #[test]
-fn debug_wasm() {
+fn assert_wasm() {
     assert_eq!(
-        std::fs::read("target/wasm32-unknown-unknown/debug/test_program.wasm").unwrap(),
```

Comment:
Am I right that you want to always push binaries to release dir, even if it had built without release flag?

Reply:
Yes, we build WASMs in release mode but the binaries path should correspond to the profile and be intuitive for the user. Also, we had an idea to cut gstd's `debug` feature when building in release profile.

Reply:
> Also, we had an idea to cut gstd's debug feature when building in release profile.

I think we need to dicuss possibility and how to do it correctly